test/functional/test-cid.js

```diff
@@ -795,7 +800,8 @@ describes.realWin('cid', {amp: true}, env => {
 
     beforeEach(() => {
       sandbox.stub(url, 'isProxyOrigin').returns(false);
-      sandbox.stub(cid.viewerCidApi_, 'isScopeOptedIn').returns('api-key');
+      ampdoc.win.document.head.innerHTML +=
+          '<meta name="amp-google-client-id-api" content="googleanalytics">';
```

Comment:
Is this change (and line 821) necessary? It's better to avoid mutating the document during tests (e.g. can break other tests if you forget to revert in `afterEach`).

I did notice that this is patterned off of test-viewer-cid-api.js. ☹️